src/nav/browser.yml

```diff
@@ -1,66 +1,98 @@
 title: Browser monitoring
 path: /docs/browser
 pages:
-  - title: Introduction to browser monitoring
+  - title: Intro to browser monitoring
     path: /docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring
-  - title: Compatibility and requirements
-    path: /docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring
-  - title: Browser agent EOL policy
-    path: /docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy
-  - title: Browser agent release notes
-    path: /docs/browser/new-relic-browser/getting-started/browser-agent-release-notes
-  - title: Guides
+  - title: Single page app (SPA) monitoring
     pages:
-      - title: Best practices for browser monitoring
-        path: /docs/browser/browser-monitoring/guides/browser-monitoring-best-practices-guide
-      - title: Guide to monitoring core web vitals
-        path: /docs/browser/new-relic-browser/guides/guide-to-core-web-vitals
+      - title: Introduction to SPA monitoring
+        path: /docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring
+      - title: SPA Compatibility and requirements
+        path: /docs/browser/single-page-app-monitoring/get-started/spa-compatibility-requirements
+      - title: Install SPA monitoring
+        path: /docs/browser/single-page-app-monitoring/get-started/install-single-page-app-monitoring
+      - title: View SPA data in UI
+        path: /docs/browser/single-page-app-monitoring/use-spa-data/view-spa-data-browser-ui
+      - title: SPA monitoring features
+        path: /docs/browser/new-relic-browser/browser-pro-features/spa-monitoring
+      - title: SPA data collection
+        path: /docs/browser/single-page-app-monitoring/use-spa-data/spa-data-collection
+      - title: Query SPA data
+        path: /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/browserspa-nrql-query-examples
+      - title: SPA APIs
+        path: /docs/browser/new-relic-browser/browser-apis/using-browser-apis#spa-api
+  - title: Best practices guide
+    path: /docs/browser/browser-monitoring/guides/browser-monitoring-best-practices-guide
+  - title: Page load timing
```

Comment:
@paperclypse I feel compelled to move these docs higher up because the "Page load timing process" page is a popular page. But I don't love how clunky the whole page tree is, especially being above the installation docs.

Reply:
@a-sassman Is there a way to change the nav title "Single page app (SPA) monitoring" to something more action oriented? "Monitor SPA pages" or something? I don't know how SPA is typically referred to...

Reply:
I'm going to change it to "Monitor single page apps (SPA)". Good suggestion!